### PR TITLE
GUVNOR-2728 : Dependency validation issues after upgrading from BxMS 6.1 to 6.3.3

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/GUVNOR-2728

### CODE REVIEW

**Comment by Rikkola** [comment]
@karreiro I could use a review, please.

File: kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/main/java/org/kie/workbench/common/screens/projecteditor/client/forms/dependencies/WhiteListCell.java [hunk 1]
```diff
@@ -67,11 +67,19 @@ public void render( Context context,
 
     }
 
-    @UiHandler( {"addAll", "addNone"} )
-    void onActionGotPressed( ClickEvent event,
+    @UiHandler({"addAll"})
+    void onAddAll( ClickEvent event,
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/main/java/org/kie/workbench/common/screens/projecteditor/client/forms/dependencies/WhiteListCell.java]: Code formatting.

File: kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/main/java/org/kie/workbench/common/screens/projecteditor/client/forms/dependencies/WhiteListColumn.java [hunk 2]
```diff
@@ -59,6 +61,12 @@ protected void showMessage( final String message ) {
 
     @Override
     public String getValue( final EnhancedDependency enhancedDependency ) {
+
+        if(whiteList.isEmpty()){
+            return ProjectEditorResources.CONSTANTS.AllPackagesIncluded();
+        }
+
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/main/java/org/kie/workbench/common/screens/projecteditor/client/forms/dependencies/WhiteListColumn.java]: Code formatting.

File: kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/test/java/org/kie/workbench/common/screens/projecteditor/client/forms/dependencies/DependencyGridTest.java [hunk 3]
```diff
@@ -199,13 +200,30 @@ public void testTogglePackagesToWhiteList() throws Exception {
 
         assertEquals( 0, whiteList.size() );
 
-        grid.onTogglePackagesToWhiteList( packages );
+        grid.onAddAll( packages );
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/test/java/org/kie/workbench/common/screens/projecteditor/client/forms/dependencies/DependencyGridTest.java]: There's a missing assert here?

**Comment by karreiro** [comment]
+1 (when green)

**Pull request merged**